setup.cfg

```diff
@@ -1,54 +0,0 @@
-[metadata]
-name = Pygments
-version = attr: pygments.__version__
-url = https://pygments.org/
-license = BSD-2-Clause
-author = Georg Brandl
-author_email = georg@python.org
-description = Pygments is a syntax highlighting package written in Python.
-long_description = file: description.rst
-long_description_content_type = text/x-rst
-platforms = any
-keywords = syntax highlighting
-classifiers =
-    Development Status :: 6 - Mature
-    Intended Audience :: Developers
-    Intended Audience :: End Users/Desktop
-    Intended Audience :: System Administrators
-    License :: OSI Approved :: BSD License
-    Operating System :: OS Independent
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-    Programming Language :: Python :: Implementation :: CPython
-    Programming Language :: Python :: Implementation :: PyPy
-    Topic :: Text Processing :: Filters
-    Topic :: Utilities
-project_urls =
-    Documentation = https://pygments.org/docs/
-    Source = https://github.com/pygments/pygments
-    Bug Tracker = https://github.com/pygments/pygments/issues
-    Changelog = https://github.com/pygments/pygments/blob/master/CHANGES
-
-[options]
-packages = find:
-zip_safe = false
```

Comment:
This and `include_package_data` doesn't seem to show up in the new one - is it the default now?